src/classes/genre.rb

```diff
@@ -0,0 +1,15 @@
+class Genre
+  attr_accessor :name
```

Comment:
Hi @brytebee, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
![Approved-d house](https://user-images.githubusercontent.com/67087664/154265067-60bc514a-d90b-43cd-8cf8-f7a31368c94e.gif)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._